### ISSUE TITLE
The question about input of SP-Net

### ISSUE THREAD

**Issue by super233** [yccyenchicheng/pytorch-SegInpaint#9]
Hi, thanks for your sharing code.
I have noticed that the label map input of SP-Net is one hot, have you try to replace it with the softmax output of Semantic FCN?
